pinot-query-runtime/src/test/resources/queries/CountDistinct.json

```diff
@@ -119,11 +119,13 @@
       {
         "comments": "table aren't actually partitioned by val thus all segments can produce duplicate results, thus [[8]]",
         "sql": "SELECT SEGMENT_PARTITIONED_DISTINCT_COUNT(val) FROM {tbl1}",
+        "ignored": true,
```

Comment:
This (and the other 2 tests in this file) started to fail as soon as I added new tests in `NullHandling.json`. I'm not sure why, but it happens when adding more queries, so it looks like an issue in the test framework.

Reply:
What exception did you get for these queries?